Example for uwerr.cf

martin-ueding opened the pull request:
In #226 I raised that the `uwerr.cf` function is broken somehow. This adds an example such that this part of the code is exercised as part of the checks and that further failures will be caught earlier.

martin-ueding commented:
I will merge that in *because* it fails such that we get a realistic representation of the state of `master`.

kostrzewa commented:
it's a typo, also keep in mind that the return value for uwerr.cf has changed, it's now a list (as documented)

martin-ueding commented:
I just saw that it was this simple, so I have pushed a fix in #228.

martin-ueding commented:
Wait, so the `plot.uwerr` resolution does not work any more because it is now just a data frame without a class?

kostrzewa commented:
`plot.uwerr` never worked on the data frame returned by `uwerr.cf`. `plot.uwerr` is for time series of a single value.

kostrzewa commented:
When I added `uwerr.cf`, I thought about returning a list of `uwerr`, such that `plot.uwerr` can be called on each time slice, but I discarded the idea as I did not fancy looking at `4*time_extent*nr_of_correls` plots :)

kostrzewa commented:
In retrospect I could have added a summary function which would produce the current output while the default would have been a nested list of `uwerr`...

martin-ueding commented:
It's fine, I just removed the two lines from the examples. I just extract the data and plot it myself with ggplot anyway. There was `tau_int` instead of `tauint` such that this column was silently dropped. I just fixed that in `master`.
